api/src/pcapi/core/offerers/models.py

```diff
@@ -359,6 +363,16 @@ def field_exists_and_has_changed(self, field: str, value: typing.Any) -> typing.
         return getattr(self, field) != value
 
 
+class VenueCriterion(PcObject, Model):
+    venueId = Column(BigInteger, ForeignKey("venue.id"), index=True, nullable=False)
+
+    venue = relationship("Venue", foreign_keys=[venueId])
+
+    criterionId = Column(BigInteger, ForeignKey("criterion.id"), nullable=False)
+
+    criterion = relationship("Criterion", foreign_keys=[criterionId])
+
+
```

Comment:
Quel intérêt de passer par une table intermédiaire? Pour une recherche plus rapide?

Reply:
C'est comme pour les offres : un tag peut avoir plusieurs lieux, et un lieu peut avoir plusieurs tags (_many-to-many_).